src/xcape/core.py

```diff
@@ -32,13 +41,21 @@ def _reshape_inputs(*args):
     return args_2d
 
 def _reshape_outputs(*args, shape=None):
+    '''
+    Function to reshape calculated output data arrays from the 1D returned format for spatial 
+    data (XYT) to a 3D-array (X,Y,T). Called by _calc_*_numpy. 
+    '''
```

Comment:
Also this is generic, so it returns to the original shape minus the Z dimensions (X1,X2,X3,X4,....Z) to (X1,X2,X3,X4,....)